content/docs/setup/kubernetes/getting-started/index.md

```diff
@@ -2,83 +2,41 @@
 title: Getting Started
 description: Download and install Istio.
 weight: 5
-aliases:
-    - /docs/setup/kubernetes/download-release/
-    - /docs/setup/kubernetes/download/
 keywords: [kubernetes]
 ---
 
-Istio offers multiple installation options depending on your platform and
-whether or not you intend to use Istio in production.
-
-- [Demo installation](/docs/setup/kubernetes/install/kubernetes/):
-   This option is ideal if you're new to Istio and just want to try it out.
-   It allows you to experiment with many Istio features with modest resource requirements.
-
-- [Custom installation with Helm](/docs/setup/kubernetes/install/helm/):
-   This option is ideal to install Istio for production use or for performance evaluation.
-
-- [Supported platform installation](/docs/setup/kubernetes/install/platform/):
-   This option is ideal if your platform provides native support for Istio-enabled clusters
-   with a [configuration profile](/docs/setup/kubernetes/additional-setup/config-profiles/)
-   corresponding to your intended use.
-
-After choosing an option and installing Istio on your cluster, you can deploy
-your own application or experiment with some of our [tasks](/docs/tasks/) and [examples](/docs/examples/).
-
 {{< tip >}}
```

Comment:
Starting the page with a tip section looks a bit strange. What do you think about removing the tip and adding the sentence "Istio can also be installed and customized in many [other ways](/docs/setup/), depending on your platform and intended use." to the end of the "If you’re new to Istio" sentence.?

Reply:
good idea